source/example-smart-app/launch.html

```diff
@@ -26,7 +26,7 @@
     <script src='./lib/js/fhir-client-cerner-additions-1.0.0.js'></script>
     <script>
       FHIR.oauth2.authorize({
-        'client_id': 'df7c5a17-52dd-4c88-8a32-cdfb557ba758',
+        'client_id': '<enter your client id here>',
```

Comment:
You may need to make the same update in the `gh-pages` branch too.
https://github.com/cerner/smart-on-fhir-tutorial/tree/gh-pages/example-smart-app

Reply:
Sure.